kernel/work.c

```diff
@@ -647,7 +647,10 @@ static void work_queue_main(void *workq_ptr, void *p2, void *p3)
 			 */
 			(void)z_sched_wake_all(&queue->drainq, 1, NULL);
 		} else {
-			; /* Final statement (see coding guideline 15.7) */
+			/* No work is available and no queue state requires
```

Comment:
Please squash this commit with the earlier one where the comment replaced here was introduced.

Reply:
Ok, I can do that. Is it to help keep the commits cleaner? jw. Thank you!

Reply:
Yes, it's better not to leave false-starts in the history within a single PR.